lom/src/pluginmgr/pluginmgr_common/pluginmgr_helper.go

```diff
@@ -173,12 +203,14 @@ func (plmgr *PluginManager) run() error {
             for {
                 serverReq, err := plmgr.clientTx.RecvServerRequest()
                 if plmgr.getShutdownStatus() {
-                    lomcommon.LogInfo("In run() RecvServerRequest: Shutdown is active, ignoring request: %v", serverReq)
-                    //return
+                    LogInfo("In run() RecvServerRequest: Shutdown is active, ignoring request: %v", serverReq)
+                    if lomcommon.GetLoMRunMode() == lomcommon.LoMRunMode_Test {
```

Comment:
Suggestion to move the logic inside for loop to a different function, if it can help to avoid the LoMRunMode_Test variable.

Reply:
This entire for loop is in goroutine and like different function.

Reply:
TODO